From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

kgremban committed:
New article for IoT Edge modbus modules

kgremban committed:
Updated intro

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

kgremban committed:
More changes

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

kgremban committed:
Added modbus to TOC

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

kgremban committed:
Walkthrough

kgremban committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nov16-modbus

kgremban committed:
Added next steps

Commit:
Added view relative latencies article.

Commit:
Fix.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into NW-Reachability

Commit:
Updates.

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

kgremban committed:
Chipalo's feedback

kgremban committed:
merge conflict

Commit:
Updates based on feedback.

Commit:
Formatting fixes.

v-nagta committed:
USER STORY 1147826 SaaS App Tutorial: Rollbar

v-nagta committed:
updated

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

kgremban committed:
Stephen's desired properties updates

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
USER STORY 1151807 SaaS App Tutorial: Zoom Tutorial Update

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

mimig1 committed:
Cosmos DB: Added note about partitioning prereqs.

mimig1 committed:
updated

curtand committed:
update screens and classic changes

curtand committed:
changes

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into classic0102

v-nagta committed:
USER STORY 1153676 SaaS App Tutorial: Amazon Web Services (AWS) Update

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
USER STORY 1153684 SaaS App Tutorial: Boomi Tutorial Update

v-viksur committed:
(ForumsTeam) Requesting to edit the spelling 

I am requesting you to edit the spelling from "reecovery" to 'recovery' in the first para of this documentation.

mimig1 committed:
updates

mimig1 committed:
updates

mimig1 committed:
update

billmath committed:
removing classic

billmath committed:
removing classic

mmacy committed:
[aci] quotas and limits

billmath committed:
removing classic portal screenshots

mmacy committed:
[aci] quotas and limits

mmacy committed:
[aci] 'per container group'

billmath committed:
updating

billmath committed:
fixing

mmacy committed:
[aci] table formatting

mmacy committed:
[aci] add Linux-only notes

PatAltimore committed:
Remove classic portal reference

PatAltimore committed:
Remove classic portal reference

PatAltimore committed:
Change author

PatAltimore committed:
Acrolinx fixes

PatAltimore committed:
Clean up metadata

mmacy committed:
[aci] integrate review comments

cwatson-cat committed:
Removing references to classic portal from Media Services topics

curtand committed:
convert from classic

curtand committed:
convert from classic

curtand committed:
missed deleting one step

curtand committed:
added ref to pic 202

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
updated

Commit:
Incorporated PM feedback.

Commit:
Removed registration info

gsilva5 committed:
Updated and corrected limitations for Linux 

moved some items out of limitations, removing the need for a service article

cherylmc committed:
Update vpn-gateway-faq-p2s-all-include.md

mmacy committed:
[aci] support URL and superscript

gsilva5 committed:
Update create-vm-accelerated-networking-cli.md

gsilva5 committed:
Update create-vm-accelerated-networking-cli.md

Commit:
jairo01

mmacy committed:
[aci] platform differences note

iainfoulds committed:
Adding Accelerated Networking docs to TOCs

cherylmc committed:
Update vpn-gateway-faq-p2s-all-include.md

cwatson-cat committed:
Added admin role for Automation topic.

cwatson-cat committed:
Fixed acrolinx score and numbering on other file.

iainfoulds committed:
Adding TOC entry querystrings

pkcsf committed:
updated the vm sku for gold durability

mmacy committed:
[aci] minor fix

rjagiewich committed:
Merge pull request #30026 from jimdial/NW-Reachability

Added view relative latencies article.

rjagiewich committed:
Merge pull request #30829 from MarkusVi/jairo01

jairo01

GitHubber17 committed:
Merge pull request #30795 from cwatson-cat/1-3-18-ClassicUpd-MediaServices

Removing references to classic portal from Media Services topics

GitHubber17 committed:
Merge pull request #30827 from cherylmc/patch-2

Update vpn-gateway-faq-p2s-all-include.md

GitHubber17 committed:
Merge pull request #30831 from cwatson-cat/1-4-2018-Automation-Ad-UserAcct

Added admin role for Automation topic.

Commit:
updated release notes for 1.14.1 and 1.14.2

Commit:
updated release notes for version 2.3.1

msmbaldwin committed:
Updated TOC to surface financial blueprint article

rjagiewich committed:
Merge pull request #30785 from PatAltimore/patricka-aad-remove-old-portal

Remove classic portal references from AAD dev articles

Commit:
Minor updates.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas0104

rjagiewich committed:
Merge pull request #30742 from v-viksur/patch-16

(ForumsTeam) Requesting to edit the spelling

kgremban committed:
Updated author for App Proxy content

MGoedtel committed:
updated article to make note of GOv Cloud

twooley committed:
updated title for SEO work

cwatson-cat committed:
Removed references to classic portal URL in cortana topic

raiye committed:
Update cloud-services-guestos-update-matrix.md

sethmanheim committed:
Fixing incorrect TOC node placement

cwatson-cat committed:
Removed stray references to classic portal from 2 mobile engagement topics

pkcsf committed:
In corporated Chacko's recommendation

iainfoulds committed:
Adding commands to list available VM SKUs

raiye committed:
Update cloud-services-guestos-msrc-releases.md

Commit:
modified date

Commit:
modified date

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov16-modbus

ktoliver committed:
Merge pull request #30446 from v-nagta/rollbar

USER STORY 1147826 SaaS App Tutorial: Rollbar

MGoedtel committed:
updated image for article

mahesh-unnikrishnan committed:
Update active-directory-ds-faqs.md

SnehaGunda committed:
Text updates

rmca14 committed:
Merge pull request #30623 from v-nagta/zoomupdate

USER STORY 1151807 SaaS App Tutorial: Zoom Tutorial Update

Commit:
Minor updates.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas0104

iainfoulds committed:
Edits and revisions

ktoliver committed:
Merge pull request #30766 from billmath/pim2

removing classic

mimig1 committed:
Incorporated feedback from abinav2307

cwatson-cat committed:
update author on Stream analytics topics so I can retire.

cjgronlund committed:
force image update + freshness

ktoliver committed:
Merge pull request #30802 from curtand/saasapp0103

convert from classic

rmca14 committed:
Merge pull request #30769 from mmacy/aci-quotas-expansion

 [aci] new: quotas and limits

Commit:
freshness

ktoliver committed:
Merge pull request #30830 from iainfoulds/vmaccelnetwork

VM TOC Accelerated Networking

rmca14 committed:
Merge pull request #30833 from pkcsf/azure-doc-pkc

updated the vm sku for gold durability

rmca14 committed:
Merge pull request #30838 from msmbaldwin/patch-1

Updated TOC to surface financial blueprint article

cjgronlund committed:
freshness update

rmca14 committed:
Merge pull request #30840 from Minewiskan/aas0104

Aas0104

HeidiSteen committed:
renamed file and added redirect entry

iainfoulds committed:
Moving commands to CLI and PoSH VM create docs

cwatson-cat committed:
Added redirects and deleted files.

ktoliver committed:
change ms.date year to 2018

change ms.date year to 2018

ktoliver committed:
add missing periods in new content

add missing periods in new content

ktoliver committed:
add missing period in new content

add missing period in new content

tamram committed:
fixing broken link

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into classic0102

mrbullwinkle committed:
Update article to remove references + screenshots to classic portal

ktoliver committed:
Merge pull request #30843 from MGoedtel/UpdateAzureVMQuick132018

updated article to make note of GOv Cloud

rjagiewich committed:
Merge pull request #30844 from kgremban/jan4-appproxyauthor

Updated author for App Proxy content

ktoliver committed:
Merge pull request #30712 from mimig1/01022018minrus

Cosmos DB: Added note about partitioning prereqs.

msjuergent committed:
thirdchange

tamram committed:
fixing broken link

kgremban committed:
Change to TOC and corrected protocol

ktoliver committed:
Merge pull request #30836 from srinathnarayanan/patch-4

updated release notes for 1.14.1 and 1.14.2

ktoliver committed:
Merge pull request #30837 from srinathnarayanan/patch-5

updated release notes for version 2.3.1

rjagiewich committed:
Update cortana-analytics-technical-guide-demand-forecast.md

mattbriggs committed:
changed to lowercase filenames

rmca14 committed:
Merge pull request #30847 from raiye/patch-1

Update cloud-services-guestos-update-matrix.md

rmca14 committed:
Merge pull request #30849 from sethmanheim/sb-tocfix

Fixing incorrect TOC node placement

rmca14 committed:
Merge pull request #30852 from raiye/patch-2

Update cloud-services-guestos-msrc-releases.md

rjagiewich committed:
Merge pull request #30851 from iainfoulds/vmazskus

Availability Zones VM SKUs

curtand committed:
new screens

rmca14 committed:
Merge pull request #30853 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-faqs.md

kgremban committed:
More app proxy author changes

tamram committed:
fixing broken link

rmca14 committed:
Merge pull request #30854 from SnehaGunda/master

Text updates

mattbriggs committed:
update cap issues

rmca14 committed:
Merge pull request #30856 from cwatson-cat/1-4-18-ClassicCln-StreamAna

Update author on Stream analytics topics so I can retire them

rjagiewich committed:
Merge pull request #30846 from cwatson-cat/1-4-18-ClassicUpd-cortana

Removed references to classic portal URL in cortana topic

GitHubber17 committed:
Merge pull request #28044 from kgremban/nov16-modbus

New article for IoT Edge modbus modules

rjagiewich committed:
Merge pull request #30733 from v-nagta/boomi

USER STORY 1153684 SaaS App Tutorial: Boomi Tutorial Update

rjagiewich committed:
Merge pull request #30731 from v-nagta/aws

USER STORY 1153676 SaaS App Tutorial: Amazon Web Services (AWS) Update

GitHubber17 committed:
Merge pull request #30845 from twooley/twtitleupdate

updated title for SEO work

Commit:
Fixing 404s and bad links

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into newFixLinks

GitHubber17 committed:
Merge pull request #30850 from cwatson-cat/1-4-18-ClassicClnUp-MobileEngagement

Removed stray references to classic portal from 2 mobile engagement...

GitHubber17 committed:
Merge pull request #30858 from Blackmist/fresh18

freshness

GitHubber17 committed:
Merge pull request #30859 from cjgronlund/fresh-jan4

Freshness update

GitHubber17 committed:
Merge pull request #30720 from curtand/classic0102

update screens and classic changes

tfosmark committed:
Merge pull request #30866 from tamram/tamram-0102

fixing broken link

gsilva5 committed:
updated future looking statements and other text around hyperthreading for clarity

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

gsilva5 committed:
Updated verbiage on hyperthreading

Ja-Dunn committed:
Merge pull request #30877 from roygara/newFixLinks

Fixing 404s and bad links

Ja-Dunn committed:
Merge pull request #30872 from kgremban/jan4-appproxyauthor

More app proxy author changes

Ja-Dunn committed:
Merge pull request #30871 from mrbullwinkle/mrb_01042018

[App Insights] Update article to remove references + screenshots to classic portal

v-alje committed:
Merge pull request #30880 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #30869 from mattbriggs/mdb_pep_cap

changed to lowercase filenames

Ja-Dunn committed:
Merge pull request #30867 from msjuergent/hlioverviewchange

thirdchange

Ja-Dunn committed:
Merge pull request #30861 from HeidiSteen/heidist-master

renamed file and added redirect entry

Ja-Dunn committed:
Merge pull request #30882 from gsilva5/patch-12

Updated verbiage on hyperthreading - Windows doc

Ja-Dunn committed:
Merge pull request #30823 from gsilva5/patch-10

Updated and corrected limitations for Linux
